packages/africa/README.md

```diff
@@ -0,0 +1,87 @@
+# 🌍 Africa
+
+[![License: CC BY 4.0](https://img.shields.io/badge/License-CC%20BY%204.0-blue.svg)](https://creativecommons.org/licenses/by/4.0/)
+
+Contains the African countries and territories listed:
+* Angola
+* Burkina Faso
+* Burundi
+* Benin
+* Botswana
+* Democratic Republic of Congo
+* Central African Republic
+* Republic of Congo
+* Côte d'Ivoire
+* Cameroon
+* Cape Verde
+* Djibouti
+* Algeria
+* Egypt
+* Western Sahara
+* Eritrea
+* Ethiopia
+* Gabon
+* Ghana
+* Gambia
+* Guinea
+* Glorioso Islands
+* Equatorial Guinea
+* Guinea-Bissau
+* Juan De Nova Island
+* Kenya
+* Comoros
+* Liberia
+* Lesotho
+* Libya
+* Morocco
+* Madagascar
+* Mali
+* Mauritania
+* Mauritius
+* Malawi
+* Mozambique
+* Namibia
+* Niger
+* Nigeria
+* Reunion
+* Rwanda
+* Seychelles
+* Sudan
+* Saint Helena
+* Sierra Leone
+* Senegal
+* Somalia
+* South Sudan
+* Sao Tome and Principe
+* Swaziland
+* Chad
+* Togo
+* Tunisia
+* Tanzania
+* Uganda
+* Mayotte
+* South Africa
+* Zambia
+* Zimbabwe
```

Comment:
Could use the alphabetical order please?

Reply:
Very well. been quite distracted. I am on it now. Thanks